recipes/libcurl/all/conanfile.py

```diff
@@ -285,9 +304,53 @@ def _configure_autotools_vars(self):
 
             del autotools_vars['LIBS']
             self.output.info("Autotools env vars: " + repr(autotools_vars))
+
+        if tools.cross_building(self.settings):
+            if self.settings.os == "iOS": 
+                ios_dev_target = str(self.settings.os.version).split(".")[0]
```

Comment:
dangerous, as `settings.os.version` is not always there. use `self.settings.get_safe("os.version")`.

Reply:
will it not 'just crash' ?
but ok, what to do if `get_safe` returns None
raise a configuration error ?

Reply:
Yes, raise a ConanInvalidConfiguration.